Documentation/DataTypes/Properties/GetText.rst

```diff
@@ -147,7 +147,12 @@ DB
    Get the value of the header field of record with uid 234 from table
    "tt\_content"::
 
-      foo = DB : tt_content:234:header
+      foo.data = DB : tt_content:234:header
+      
+   Get the value of the header field of a record, whose uid is stored in a GET
+   parameter `myContentId`::
+   
+      foo.data.dataWrap = DB : tt_content:{GP : myContentId}:header
```

Comment:
are we sure that this is not an SQL injection? From the top of my head I'm unsure if that's validated to int internally or not.

Reply:
Hehe, see my comment below. ;-)

Reply:
ha, perfect :) thanks.